backend/resources/order.py

```diff
@@ -0,0 +1,44 @@
+# [Flask]
+from flask import make_response, request, jsonify
+from flask_restplus import Namespace, Resource
+from flask_jwt import jwt_required
+from flask_api.status import *
+
+# [App]
+from backend.models.users import OrdersModel
+
+
+api = Namespace('orders', description='Orders related operations')
+
+
+@api.route('/<pk>', methods=['POST'])
+class OrdersDetail(Resource):
+    """ post an order  """
+
+    def post(self):
+        
+        post_data = request.get_json()
+        
+        # Query to see if the order already exists
+        order = OrdersModel.find_by_id(id=post_data.get('id', None))
```

Comment:
Since id is automatically generated by the model, I, as a user, should not have to pass the ID in the request data. If we do want to ensure an order is not duplicated, then a unique_identifier field should be added. In any model, id should be automatically generated.